app/services/foreman_templates/action.rb

```diff
@@ -22,17 +22,6 @@ def initialize(args = {})
       assign_attributes args
     end
 
-    def get_default_branch(repo)
```

Comment:
It is not related to the response format, but the method relies on the naming that we have in our repositories. I removed it because I already make changes in `TemplateExporter#export!` and I see no value in keeping it.